challenge_speech_recognition/src/indirect_speech_recognition.py

```diff
@@ -65,9 +65,9 @@ def _turn_to_closest_entity(self):
 
         # Turn towards the operator
         current = self.robot.base.get_location()
-        robot_th = tf.euler_z_from_quaternion(current.pose.orientation)
-        desired_th = math.atan2(operator.pose.position.y - current.pose.position.y,
-                                operator.pose.position.x - current.pose.position.x)
+        robot_th = current.M.GetRPY[2]  # Get the Yaw, rotation around Z
```

Comment:
Can you apply your review to the correct PR at #163?
I wrongly said this branch should go into master but it should go into refactor/cleanup_and_merge